doc/locale/ja/LC_MESSAGES/admin-guide/configuration/cache-basics.en.po

```diff
@@ -530,6 +544,10 @@ msgid ""
 "information to users on demand. It improves performance and frees up "
 "Internet bandwidth for other tasks."
 msgstr ""
+"HTTPプロキシーキャッシュは頻繁にアクセスされる（ドキュメント、イメージや記事"
```

Comment:
Please add a space after `HTTP`? Sorry for bother you, but we have a rule to add a space before and after english words. Details are in https://github.com/trafficserver-doc-ja/trafficserver-doc-ja/issues/12